BinaryOptionsToolsV2/BinaryOptionsToolsV2/pocketoption/asynchronous.py

```
@@ -0,0 +1,654 @@
+        """
+        if config is not None:
+            if isinstance(config, dict):
+                self.config = Config.from_dict(config)
+            elif isinstance(config, str):
+                self.config = Config.from_json(config)
+            elif isinstance(config, Config):
+                self.config = config
+            else:
+                raise ValueError(
+                    "Config must be either a Config object, dictionary, or JSON string"
+                )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)